src/app/pages/send/send-crypto-asset-form/form/stacks-sip10/sip10-token-send-form-container.tsx

```diff
@@ -49,7 +49,7 @@ export function Sip10TokenSendFormContainer({
         )
       }
       name={symbol}
```

Comment:
I don't think you need this `toUpperCase(` as it's done in `<SelectedAsset` and passed as uppercase anyway

Reply:
Yeah, I did need to for some reason in the UI when I was refactoring to reuse the component ...does seem weird so will double check, maybe I just forgot to remove.

Reply:
I was doing some refactoring to this app wide so don't worry, I can take care of them